src/components/AppNavigation/EditCalendarModal.vue

```diff
@@ -24,7 +24,19 @@
 					:placeholder="$t('calendar', 'Calendar name …')"
 					@input="calendarNameChanged = true">
 			</div>
-
+			<template v-if="canBeShared">
+				<NcListItem :name="$t('calendar', 'All events in this calendar show you as busy by default')"
```

Comment:
This is not within a list and thus will be invalid HTML. Only use `NcListItem` within `NcAppNavigationList` or a plain `ul` or `ol` tag.

This also is accessible wise really bad, besides the list item being not in a list you are using a link as checkbox without setting the role correctly.
Instead I would recommend just using `NcCheckboxRadioSwitch` with checkbox type and overwrite the color of the checkbox (we could make this also an option in the library but also works with styles here).